packages/nimble-components/src/table/template.ts

```diff
@@ -38,95 +38,112 @@ export const template = html<Table>`
-                                <${tableHeaderTag}
-                                    class="header"
-                                    sort-direction="${x => (typeof x.columnInternals.currentSortIndex === 'number' ? x.columnInternals.currentSortDirection : TableColumnSortDirection.none)}"
-                                    ?first-sorted-column="${(x, c) => x === c.parent.firstSortedColumn}"
-                                    @click="${(x, c) => c.parent.toggleColumnSort(x, (c.event as MouseEvent).shiftKey)}"
-                                    :isGrouped=${x => (typeof x.columnInternals.groupIndex === 'number' && !x.columnInternals.groupingDisabled)}
-                                >
-                                    <slot name="${x => x.slot}"></slot>
-                                </${tableHeaderTag}>
-                            `)}
-                        `)}
-                        <div class="header-scrollbar-spacer"></div>
-                    </span>
-                </div>
-            </div>
-            <div class="table-viewport" ${ref('viewport')}>
-                <div class="table-scroll"></div>
-                <div class="table-row-container" ${children({ property: 'rowElements', filter: elements(tableRowTag) })}
-                     role="rowgroup">
-                    ${when(x => x.columns.length > 0 && x.canRenderRows, html<Table>`
-                        ${repeat(x => x.virtualizer.visibleItems, html<VirtualItem, Table>`
-                            ${when((x, c) => (c.parent as Table).tableData[x.index]?.isGrouped, html<VirtualItem, Table>`
-                                <${tableGroupRowTag}
-                                    class="group-row"
-                                    :groupRowValue="${(x, c) => c.parent.tableData[x.index]?.groupRowValue}"
-                                    ?expanded="${(x, c) => c.parent.tableData[x.index]?.isExpanded}"
-                                    :nestingLevel="${(x, c) => c.parent.tableData[x.index]?.nestingLevel}"
-                                    :leafItemCount="${(x, c) => c.parent.tableData[x.index]?.leafItemCount}"
-                                    :groupColumn="${(x, c) => c.parent.tableData[x.index]?.groupColumn}"
-                                    ?selectable="${(_, c) => c.parent.selectionMode === TableRowSelectionMode.multiple}"
-                                    selection-state="${(x, c) => c.parent.tableData[x.index]?.selectionState}"
-                                    @group-selection-toggle="${(x, c) => c.parent.onRowSelectionToggle(x.index, c.event as CustomEvent<TableRowSelectionToggleEventDetail>)}"
-                                    @group-expand-toggle="${(x, c) => c.parent.handleGroupRowExpanded(x.index, c.event)}"
-                                >
-                                </${tableGroupRowTag}>
+            <div class="glass-pane" style="
```

Comment:
This `div` is responsible for preventing hover states on elements during a column sizing operation. I'm not sure of a cleaner way to provide this behavior. Additionally, this proved difficult to write a unit test for, as I believe we ultimately need an API that actually moves a mouse to a position on screen.

Open to a class name other than `glass-pane` as well.